msgservice/aggregatebytime_test.go

```diff
@@ -52,9 +52,15 @@ func TestAggregateByTimeout(t *testing.T) {
 	srvUrl := ""
 
 	srv1 := new(MsgService)
-	srv1.MsgHandling([]byte(mockScan1), demoEmailPlg, demoRoute, demoInptEval, &srvUrl)
-	srv1.MsgHandling([]byte(mockScan2), demoEmailPlg, demoRoute, demoInptEval, &srvUrl)
-	srv1.MsgHandling([]byte(mockScan3), demoEmailPlg, demoRoute, demoInptEval, &srvUrl)
+	if srv1.EvaluateRegoRule(demoRoute, []byte(mockScan1)) {
```

Comment:
`Route` instance here has no OPA filter defined. So no need to wrap `MsgHandling` by `if`

Reply:
Done